bridetalk_project/src/components/molecule/forms/createuserform/CreateUserForm.jsx

```diff
@@ -0,0 +1,71 @@
+import {
+  Headline1,
+  EnterText,
+  EnterEmail,
+  EnterPassword,
+  DropdownLocation,
+  PrimaryButton,
+  CancelButton,
+} from "../../../atoms";
+
+import "./createuserform.css";
+import { Link } from "react-router-dom";
+
+import { useNavigate } from "react-router-dom";
+export const CreateUserForm = () => {
+  let navigate = useNavigate();
+
+  const toFrontPage = () => {
+    let path = "/";
+    navigate(path);
+  };
+
+  const toLoginPage = () => {
+    let path = "/login";
+    navigate(path);
+  };
+
+  const goBack = () => {
+    let path = navigate(-1);
+    navigate(path);
+  };
+
+  return (
+    <div>
+      <div className="form">
+        <Headline1 headline="Sign up to Bride Talk"></Headline1>
+        <div
+          handleClick={toLoginPage}
+          className="question"
+          style={{
+            paddingBottom: "15px",
```

Comment:
Why the inline-styling since you have a classname that you can target in CSS? Same with the classname="login" below.